### ISSUE TITLE
(Other) Real-World, Large-Scale Dask Uses

### ISSUE THREAD

**Issue by Scusemua** [pangeo-data/pangeo#574]
Hello all,

I'm a computer science student currently conducting research on the elasticity and scalability of distributed data analytics frameworks (especially [Dask](https://github.com/dask/dask)) on container clouds.

I am wondering if anybody could please point me to some real-world large-scale applications using Dask in the general fields of computational sciences, data sciences, data analytics, high-performance computing, etc. 

Specifically, I am looking for applications I can use to test the scalability of Dask on Kubernetes, especially on a medium to large scale. I am fairly new to this area and I would greatly appreciate any assistance you can provide in directing me to such applications. 

I have discovered the "[Dask Stories](https://dask-stories.readthedocs.io/en/latest/)" section on the Dask docs, also. These are essentially the type of things that I'm looking for.

**Comment by rabernat** [comment]
Hi @Scusemua - If you're not looking specifically for applications from Earth sciences, you are probably best off pursuing this question on the dask repo. Several of us here have already contributed to the "dask stories" collection.

**Comment by mrocklin** [comment]
@Scusemua actually contacted me directly asking for information and I pointed him here.  Pangeo is generally a good example of a computational science doing large scale data analytics with Dask on Kubernetes.  I figured that he might enjoy links to some of the binders that the Pangeo community has created, both as an example of how Dask works on Kubernetes, and as examples of the kind of science that is done today.
